cmd/lakefs/cmd/run.go

```diff
@@ -83,6 +84,15 @@ var runCmd = &cobra.Command{
 			logger.WithError(err).Fatal("Failed to create block adapter")
 		}
 
+		//  Migrate old MVCC - if cataloger type is not set,
+		//  warn the user in case migrate didn't run and there are MVCC repositories
+		if cfg.GetCatalogerType() == "" {
+			migrationRequired := catalogmigrate.CheckMigrationRequired(dbPool)
+			if migrationRequired {
+				logger.Fatal("Migration of old data is required! (lakefs migrate db)")
```

Comment:
please link to the the future release notes that explain the migration steps

Reply:
Update https://docs.lakefs.io/deploying/upgrade.html?
or
Link https://github.com/treeverse/lakeFS/releases/tag/v0.30.0

Reply:
Added link to the docs in the code and detailed explanation by Yoni.
The release notes will include the same information.